webpack/components/extensions/HostDetails/Tabs/ContentTab.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import RoutedTabs from '../../../../components/RoutedTabs';
+import EmptyPage from 'foremanReact/components/common/EmptyState/EmptyStatePattern';
+import { Tabs, Tab, TabTitleText} from '@patternfly/react-core';
+
+
+const ContentTab = () => {
+  const [activeTab, setActiveTab] = useState('Content');
+  const tabs = [
```

Comment:
This can be moved above the function scope, as it is a constant that doesn't use state.